src/HazelcastClient.ts

```diff
@@ -365,24 +452,27 @@ export class HazelcastClient {
     }
 
     /** @internal */
-    getProxyManager(): ProxyManager {
-        return this.proxyManager;
-    }
-
-    /** @internal */
-    getNearCacheManager(): NearCacheManager {
-        return this.nearCacheManager;
+    getClusterService(): ClusterService {
+        return this.clusterService;
     }
 
     /** @internal */
-    getClusterService(): ClusterService {
-        return this.clusterService;
+    getConnectionRegistry(): ConnectionRegistry {
```

Comment:
This is unused should we keep this

Reply:
Nope, we should remove this one as all `ConnectionRegistry` consumers already have it.

Reply:
We wont need it in tests?

Reply:
It should be fine to access the such infrequently refactored fields directly in tests (as they're written in JS, so `private` modifier is not an issue there).